proto/minder/v1/minder.proto

```diff
@@ -1230,7 +1250,8 @@ message UpstreamRepositoryRef {
             pattern: "^[-.[:alnum:]_]+$",
             max_len: 200,
         },
-        (buf.validate.field).ignore = IGNORE_IF_DEFAULT_VALUE
+        (buf.validate.field).ignore = IGNORE_IF_DEFAULT_VALUE,
+        (google.api.field_behavior) = REQUIRED
```

Comment:
I'm confused by this. We say "IGNORE_IF_DEFAULT_VALUE" then check for optional?

Reply:
I'm confident that the `IGNORE_IF_DEFAULT_VALUE` isn't correct because when processing the request we fail if `name` is unset. I would like to do some additional testing and remove that annotation in a different PR.